src/Form/Extension/DeprecatedCollectionTypeExtension.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of the Sonata Project package.
+ *
+ * (c) Thomas Rabaix <thomas.rabaix@sonata-project.org>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sonata\DoctrinePHPCRAdminBundle\Form\Extension;
+
+/**
+ * @deprecated
+ * Extend the sonata collection type to sort the collection so the reordering
+ * is automatically persisted in phpcr-odm.
+ */
+class DeprecatedCollectionTypeExtension extends CollectionTypeExtension
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function getExtendedType()
+    {
+        return self::getExtendedTypes()[1];
```

Comment:
I don't think if we can trust directly on the offset set in the parent class (this class is not marked as final). Is there a reason to avoid the usage of the static value here?